### PR TITLE
[full-ci] Bump ODS to v10.0.0-rc2

### PR DESCRIPTION
**Pull request by pascalwengerter**
## Description
We need this meta-PR since there's two breaking changes in the ODS which are adressed in two separate PRs in web. Let's unite them here if they're good regarding linting, unit tests and code review and then get this one here merged back to master once CI is green.

### CODE REVIEW

**Comment by pascalwengerter** [comment]
@paulcod3 why did you close this one?

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait1</strong> https://drone.owncloud.com/owncloud/web/18922/42/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:59

**Comment by paulcod3** [comment]
@pascalwengerter i didn't o.O

**Comment by pascalwengerter** [comment]
> @pascalwengerter i didn't o.O

Happened automatically due to some rebasing and the PR eventually having 0 commits (and GitHub autoclosing^^), all good

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18943/12/1
The following scenarios passed on retry:
- webUIFavorites/favoritesFile.feature:85

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18947/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersToRootSharingIndicator/shareWithUsers.feature:98

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18951/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:154

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternalRoot</strong> https://drone.owncloud.com/owncloud/web/18953/40/1
The following scenarios passed on retry:
- webUISharingExternalToRoot/federationSharing.feature:356

**Comment by ownclouders** [comment]
Results for <strong>oC10MoveUpload</strong> https://drone.owncloud.com/owncloud/web/18953/14/1
The following scenarios passed on retry:
- webUIMoveFilesFolders/moveFiles.feature:20

File: packages/web-app-files/src/views/LocationPicker.vue [hunk 1]
```diff
@@ -323,7 +324,7 @@ export default {
 
     leaveLocationPicker(target) {
       if (this.isPublicContext) {
-        this.$router.push({ name: 'files-public-list', params: { token: target } })
+        this.$router.push({ name: 'files-public-list', params: { item: target } })
```
Comment (pascalwengerter) [inline packages/web-app-files/src/views/LocationPicker.vue:327]: @kulmann is that change fine?

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternal</strong> https://drone.owncloud.com/owncloud/web/18960/39/1
The following scenarios passed on retry:
- webUISharingExternal/federationSharing.feature:363

File: tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature [hunk 2]
```diff
@@ -79,7 +79,7 @@ Feature: accept/decline shares coming from internal users
     And user "Brian" has been added to group "grp1"
     And user "Alice" has shared folder "/simple-folder" with group "grp1"
     And user "Alice" has shared folder "/testimage.jpg" with user "Brian"
-    When the user browses to the files page
+    When the user has browsed to the files page
```
Comment (phil-davis) [inline tests/acceptance/features/webUISharingAcceptSharesToRoot/acceptShares.feature:82]:
```suggestion
    And the user has browsed to the files page
```

If we use the "has done..." form then it fits as a `Given` step, so use `And`here. This scenario is a bit "naughty" because it tests 2 things - that the auto-accept works (the shares are immediately visible) and that shares appear on the shared-with-me page. So it had 2 `When`/`Then` sets of steps. Alternatively, you could revert this change.

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsersSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/18965/28/1
The following scenarios passed on retry:
- webUISharingInternalUsersSharingIndicator/shareWithUsers.feature:154

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPublicManagement</strong> https://drone.owncloud.com/owncloud/web/18965/34/1
The following scenarios passed on retry:
- webUISharingPublicManagement/publicLinkIndicator.feature:27

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/18965/50/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:56

**Comment by ownclouders** [comment]
Results for <strong>oC10IntegrationApp1</strong> https://drone.owncloud.com/owncloud/web/18965/65/1
The following scenarios passed on retry:
- webUIFilesCopy/copy.feature:36

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18965/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Comment by ownclouders** [comment]
Results for <strong>oC10Sharing1</strong> https://drone.owncloud.com/owncloud/web/18967/16/1
The following scenarios passed on retry:
- webUISharingAutocompletion/shareAutocompletionSpecialChars.feature:74

**Comment by ownclouders** [comment]
Results for <strong>oC10Files1</strong> https://drone.owncloud.com/owncloud/web/18967/12/1
The following scenarios passed on retry:
- webUIFilesActionMenu/versions.feature:36

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternal</strong> https://drone.owncloud.com/owncloud/web/18967/39/1
The following scenarios passed on retry:
- webUISharingExternal/federationSharing.feature:363

**Comment by ownclouders** [comment]
Results for <strong>oC10XGAPortrait2</strong> https://drone.owncloud.com/owncloud/web/18989/43/1
The following scenarios passed on retry:
- webUIMoveFilesFolders/moveFiles.feature:51

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingPublicManagement</strong> https://drone.owncloud.com/owncloud/web/18991/34/1
The following scenarios passed on retry:
- webUISharingPublicManagement/publicLinkIndicator.feature:27

**Review by kulmann — APPROVED** [review]
